### ISSUE TITLE
Building not rendered

### ISSUE THREAD

**Issue by scaidermern** [osmandapp/Osmand#4488]
OsmAnd doesn't render this building: https://www.openstreetmap.org/way/445375600

![photo_2017-09-26_21-50-56](https://user-images.githubusercontent.com/1040784/30881062-cf0c0d5e-a304-11e7-980d-ee0e4c9d93f8.jpg)

In the screenshot the building is located at the [post box](https://www.openstreetmap.org/node/4425042396). It shares a node with this post box and also with a [cliff](https://www.openstreetmap.org/way/248072229).

The building has been added 12 months ago (I changed some details, so the latest version is pretty new).

**Comment by naoliv** [comment]
Maybe it's small and was removed during the simplification process?

**Comment by scaidermern** [comment]
I don't think so. It has a perimeter of 9.8 m whereas the [building](https://www.openstreetmap.org/way/445375601) that is slightly visible at the top has a perimeter of only 8.4 m and gets rendered nevertheless.

**Comment by xmd5a2** [comment]
Checked this. Yes, this building is filtered by area.

**Comment by scaidermern** [comment]
> Yes, this building is filtered by area.

What does this mean, can you elaborate? Is this building dropped because it is too small?

If so there is still a bug. This building is also tagged as tourism=information and information=office. So you are basically dropping a POI. Either keep such buildings if they contain other important tags. Or convert them to a POI node.

Please reopen this bug.

**Comment by xmd5a2** [comment]
Building is dropped only from map data. But yes, it is not possible to select POI directly from map.
![screenshot_2017-09-28-22-52-10](https://user-images.githubusercontent.com/5829014/30987551-1ea6e2f2-a4a0-11e7-9de3-a96bb2eb2197.png)

**Comment by scaidermern** [comment]
Oh, it is still searchable. Haven't tried that. This is good to know.
